src/nni_manager/training_service/pai/paiTrainingService.ts

```diff
@@ -436,7 +436,20 @@ class PAITrainingService implements TrainingService {
                 this.paiTrialConfig.shmMB,
             )
         ];
+        // Upload authFile to hdfs
+        let authFileHdfsDir: string | undefined = undefined;
```

Comment:
I think this logic should be included in setclustermetadata, but this field is within trialconfig....

Reply:
fixed.